bob-common/src/bob_client.rs

```diff
@@ -40,9 +41,18 @@ pub mod b_client {
             node: Node,
             operation_timeout: Duration,
             metrics: BobClientMetrics,
+            ca_cert_path: Option<&str>,
```

Comment:
I think, it is better to pass loaded bytes of certificate. This will avoid the problem of replacement certificate file in runtime. 

Also, I suggest to make this argument the last one